draft-trammell-rains-protocol.md

```diff
@@ -593,24 +716,31 @@ and notification maps is given in the symbol table below:
 
 {: #tabmkey title="CBOR Map Keys used in RAINS"}
 
-| Code | Name           | Description                                   |
-|-----:|----------------|-----------------------------------------------|
-| 0    | signatures     | Signatures on a message or section            |
-| 1    | capabilities   | Capabilities of server sending message        |
-| 2    | token          | Token for referring to a data item            |
-| 3    | subject-name   | Subject name in an assertion, shard or zone   |
-| 4    | subject-zone   | Zone name in an assertion, shard or zone      |
-| 5    | subject-addr   | Subject address in address assertion          |
-| 6    | context        | Context of an assertion, shard, zone or query |
-| 7    | objects        | Objects of an assertion                       |
-| 8    | query-name     | Fully qualified name for a query              |
-| 10   | query-types    | Acceptable object types for query             |
-| 11   | shard-range    | Lexical range of Assertions in Shard          |
-| 12   | query-expires  | Absolute timestamp for query expiration       |
-| 13   | query-opts     | Set of query options requested                |
-| 21   | note-type      | Notification type                             |
-| 22   | note-data      | Additional notification data                  |
-| 23   | content        | Content of a message, shard, or zone          |
+| Code | Name           | Description                                          |
+|-----:|----------------|------------------------------------------------      |
```

Comment:
consider whether any of these should be >23

Reply:
Why did you chose this range? "Integer map keys in the range -22 to +23
are reserved for the use of future versions or extensions to the RAINS protocol"
We have about 25 entries left. Is that enough?